ui/lib/core/addon/components/replication-dashboard.js

```diff
@@ -8,14 +8,35 @@ export default Component.extend({
   data: null,
   replicationDetails: null,
   isSecondary: null,
-  dr: null,
   isSyncing: computed('replicationDetails', 'isSecondary', function() {
     const { state } = this.replicationDetails;
     const isSecondary = this.isSecondary;
     return isSecondary && state && clusterStates([state]).isSyncing;
   }),
-  isReindexing: computed('data', function() {
-    // TODO: make this a real value
-    return false;
+  isReindexing: computed('replicationDetails', function() {
+    const { replicationDetails } = this;
+    return !!replicationDetails.reindex_in_progress;
+  }),
+  reindexingStage: computed('replicationDetails', function() {
+    const { replicationDetails } = this;
+    const stage = replicationDetails.reindex_stage;
+    // specify the stage if we have one
+    if (stage) {
+      return `: ${stage}`;
+    }
+    return '';
+  }),
+  reindexingProgress: computed('replicationDetails', function() {
+    // TODO: use this value to display a progress bar
+    const { replicationDetails } = this;
+    const { reindex_building_progress, reindex_building_total } = replicationDetails;
```

Comment:
Small nit: these two lines can be combined like: `const { reindex_building_progress, reindex_building_total } = this.replicationDetails;`